src/test/resources/callexistingmethod/expected/com/example/Simple.java

```diff
@@ -0,0 +1,10 @@
+package com.example;
+import java.util.Random;
+
+public class Simple {
+    public void test() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
We should discuss this case in our meeting tomorrow. I see two options:
* generate + populate a source file for `java.util.Random`, and don't link against the Java standard library
* generate + populate a stub file for `java.util.Random`, and continue to link against the standard library

The current spec doesn't fully require the latter, but I think it's the right thing to do exactly for classes that are in the JDK. For non-JDK classes that aren't from the given source code, I expect we'll want to do the former (so we'll need to support both eventually).

Reply:
To motivate this a bit further, we're planning to use this to try out different combinations of annotations on the dependencies of the target method, so the stub file would be created specifically for that. We can leave that as a TODO for now.